simphony_ui/liggghts_model.py

```diff
@@ -0,0 +1,57 @@
+from traits.api import HasStrictTraits, Int, Float, Enum, Array, File
+from traitsui.api import View, Item, VGroup, HGroup
+
+
+class LiggghtsModel(HasStrictTraits):
+    """ The model of Liggghts input parameters """
+
+    #: The duration of a step of computation.
+    timestep = Float(1e-6)
+
+    #: The number of iterations in the simulation.
+    num_iterations = Int(10)
+
+    #: The input file used for liggghts.
+    input_file = File()
+
+    boundary_condition_x = Enum('periodic', 'fixed', 'shrink-wrapped')
+    boundary_condition_y = Enum('periodic', 'fixed', 'shrink-wrapped')
+    boundary_condition_y.default_value = 'fixed'
```

Comment:
Ok, in this case, I would leave them in the same order, but I would simply use a _boundary_condition_y_default() method instead.